global-news-app/src/assets/styles/styled.d.ts

```diff
@@ -4,7 +4,7 @@ declare module "styled-components" {
   export interface DefaultTheme {
     colors: {
       white: string;
-      bodyBackground:string;
```

Comment:
make sure that you have format on save enabled

![Screenshot 2024-03-13 at 13 04 22](https://github.com/idan-gute/global-news-app/assets/104825246/345fdd6a-2d55-4526-9263-7116f9296d1e)